File: exercicios/Python/1132/1132.py
soma = 0
v = int(input())
v2 = int(input())

if v2>v:
    for n in range(v,(v2+1)):
        if(n%13 != 0):
            soma += n

if v > v2:
    for n in range(v2,(v+1)):
        if n%13 != 0:
            soma += n

print(soma)

File: exercicios/JavaScript/1132/1132.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var x = parseInt(lines.shift());
var y = parseInt(lines.shift());

var menor = x<y ? x : y;
var maior = x>y ? x : y;

var soma=0;

for(let i=menor; i<=maior; i++){
    if(i % 13 !== 0){
        soma+=i;
    }
}

console.log(soma);